modules/OsmOsisManager.py

```diff
@@ -30,6 +30,8 @@
 import shutil
 import datetime
 import time
+import osmium
+import dateutil
```

Comment:
Est-ce que ça ne vaudrait pas le coup de copier en OsmiumManager, et garder OsmOsisManager ?

Reply:
No, OsmOsisManager is mainly about the osmosis database, and we keep if